code/datums/components/chem_booster.dm

```diff
@@ -199,8 +211,11 @@
 	update_boost(boost_amount*2, FALSE)
 	power_action.action_icon_state = "cboost_on"
 	power_action.update_button_icon()
-	to_chat(wearer, "<span class='notice'>Commensing reagent injection.</span>")
 	playsound(get_turf(wearer), 'sound/effects/bubbles.ogg', 30, 1)
+	to_chat(wearer, "<span class='notice'>Commensing reagent injection.<b>[(automatic_meds_use && meds_beaker.reagents.total_volume) ? " Adding additional reagents." : ""]</b></span>")
+	if(automatic_meds_use)
+		meds_beaker.reagents.trans_to(wearer, 60)
+		scan_user()
```

Comment:
Does this really need to open a popup when you turn it on

Reply:
replaced scan with chat message saying what reagents were injected